### PR TITLE
Introduced NewApplicationPaymentsTest

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1251

### Change description ###
This is an initial version of the test, there will be more checks according the task.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/ZipFileHelper.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public static String randomFileName() {
             }
 
             // add metadata
-            zos.putNextEntry(new ZipEntry("metadata.json"));
+            zos.putNextEntry(new ZipEntry("test-data/new-application-payments/metadata.json"));
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/ZipFileHelper.java:76]: ??

Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/ZipFileHelper.java:76]: `new ZipEntry("metadata.json")` is not loading a resource, it's just creating an entry with name specified as parameter.

Comment (apereverzin) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/ZipFileHelper.java:76]: Smart intellij

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/NewApplicationPaymentsTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        // create zip file
+        var zipArchive = ZipFileHelper.createZipArchive(
+            singletonList("test-data/new-application-payments/1111002.pdf"),
+            "test-data/new-application-payments/metadata.json",
+            zipFileName
+        );
+
+        // upload zip file
+        StorageHelper.uploadZipFile("bulkscan", zipFileName, zipArchive);
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/NewApplicationPaymentsTest.java:24]: I'd say both comments can be removed. They'll literally methods names that are being called

**Pull request merged**